### PR TITLE
Add split_bib to bs4_book

### PR DESCRIPTION
**Pull request by shirdekel**
- From https://github.com/rstudio/bookdown/issues/1146

### CODE REVIEW

File: R/bs4_book.R [hunk 1]
```diff
@@ -48,7 +48,8 @@ bs4_book <- function(
                      ...,
                      lib_dir = "libs",
                      pandoc_args = NULL,
-                     extra_dependencies = NULL
+                     extra_dependencies = NULL,
+                     split_bib = split_bib
```
Comment (cderv) [inline R/bs4_book.R:52]: This should be FALSE by default right ? 
```suggestion
                     split_bib = FALSE
```

Also when adding an argument, it should be added to the documentation here: 
https://github.com/rstudio/bookdown/pull/1185/files#diff-ff358cd3f877c575281978ccf49a2c8869cdcef59cdba125698dc3e8966a570aL41

**Review by cderv — CHANGES_REQUESTED** [review]
This seems like a simple change. 

Did you test it against your book ? Did you find any conflict with other features ? 

Can you add a NEWS bullet too ? 

As you added a new argument, you need to document so the CI tests do not error on doc mismatch. thanks !

**Comment by shirdekel** [comment]
Looks like there's still an issue with the documentation. I tried looking at the `gitbook.R` docs but it's not documented there, only in `html.R`.  Should I give it its own `@param` description as in `html.R`?

**Comment by cderv** [comment]
Ok I'll have a look at the issue and do it. Thanks !

**Review by cderv — APPROVED** [review]
I have made the change. `split_bib = TRUE` will now be possible to specify in `bs4_book()`. 

This is useful when no CSL with footnote style for citations is used. 
https://pkgs.rstudio.com/bookdown/reference/bs4_book.html#references-bibliography
In that case, `split_bib = TRUE` will do nothing.

When a CSL with footnote style is used, citations are set inline in a linked popup showing on click. This has the same intent a putting the references at the end of the chapter but using a different way to do it.

**Pull request merged**